Add products or services providers to adopters page (#316)
tikv/website: content/adopters.md

|[KuGou](https://en.wikipedia.org/wiki/KuGou)|Entertainment and Music Streaming|
|[Meitu](https://en.wikipedia.org/wiki/Meitu)|Photo-editing|

The following is a list of third-party companies and individuals who provide products or services related to TiKV. The list is in alphabetical order:

> * [TiDB Cloud](https://tidbcloud.com)

Reviewer: ```suggestion
| Company | Industry  |
| :--- | :--- | 
| [TiDB Cloud](https://tidbcloud.com)| DBaaS|
```

|[KuGou](https://en.wikipedia.org/wiki/KuGou)|Entertainment and Music Streaming|
|[Meitu](https://en.wikipedia.org/wiki/Meitu)|Photo-editing|

> Commercial Support/Offering upon TiKV
> 
> This is a list of third-party companies and individuals who provide products or services related to TiKV. The list is in alphabetical order:
> 
> * [TiDB Cloud](https://tidbcloud.com)

Reviewer: ```suggestion
The following is a list of third-party companies and individuals who provide products or services related to TiKV. The list is in alphabetical order:

* [TiDB Cloud](https://tidbcloud.com)
```

> | Company | Industry |
> | :--- | :--- |
> | [TiDB Cloud](https://tidbcloud.com)| DBaaS |

Reviewer: ```suggestion
| Company | Offering | Industry |
| :--- | :--- |:--- |
|PingCAP| [TiDB Cloud](https://tidbcloud.com)| DBaaS |
```
